OpenChange/MAPIStoreMailMessage.m

```diff
@@ -537,6 +537,17 @@ - (void) _fetchBodyData
     }
 
   bodySetup = YES;
+  /* Free data from header setup that it is only used for setup the body */
+  [bodyContentKeys release];
+  bodyContentKeys = nil;  
```

Comment:
I'd say there is no need to set to nil (on `dealloc` method `[nil release]` will be called but that's valid and won't do anything)